code/modules/clothing/neck/collar_bomb.dm

```diff
@@ -0,0 +1,120 @@
+///Special neckwear that kills its wearer if triggered, by either its specific remote or assemblies.
+/obj/item/clothing/neck/collar_bomb
+	name = "collar bomb"
+	desc = "A cumbersome collar of some sort, filled with just enough explosive to rip one's head off... at least that's what it reads on the front tag."
+	icon_state = "collar_bomb"
+	icon = 'icons/obj/clothing/neck.dmi'
+	inhand_icon_state = "reverse_bear_trap"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	clothing_flags = INEDIBLE_CLOTHING
+	armor_type = /datum/armor/collar_bomb
+	equip_delay_self = 6 SECONDS
+	equip_delay_other = 8 SECONDS
+	///The button it's associated with
+	var/obj/item/collar_bomb_button/button
+	///Whether the collar countdown has been triggered.
+	var/active = FALSE
+
+/datum/armor/collar_bomb
+	fire = 97
+	bomb = 97
+	acid = 97
+
+/obj/item/clothing/neck/collar_bomb/Initialize(mapload, obj/item/collar_bomb_button/button)
+	. = ..()
+	src.button = button
+	button?.collar = src
+	set_wires(new /datum/wires/collar_bomb(src))
+
+/obj/item/clothing/neck/collar_bomb/Destroy()
+	button?.collar = null
+	button = null
+	return ..()
+
+/obj/item/clothing/neck/collar_bomb/examine(mob/user)
+	. = ..()
+	if(user.get_item_by_slot(ITEM_SLOT_NECK) == src)
+		return
+	. += span_tinynotice("It has a [EXAMINE_HINT("wire")] panel that could be interacted with...")
+
+/obj/item/clothing/neck/collar_bomb/attackby(obj/item/item, mob/user, params)
+	if(is_wire_tool(item))
+		wires.interact(user)
+	else
+		return ..()
+
+/obj/item/clothing/neck/collar_bomb/equipped(mob/user, slot, initial = FALSE)
+	. = ..()
+	if(slot == ITEM_SLOT_NECK)
+		ADD_TRAIT(src, TRAIT_NODROP, INNATE_TRAIT)
+
+/obj/item/clothing/neck/collar_bomb/dropped(mob/user, silent = FALSE)
+	. = ..()
+	REMOVE_TRAIT(src, TRAIT_NODROP, INNATE_TRAIT)
+
+/obj/item/clothing/neck/collar_bomb/proc/explosive_countdown(ticks_left)
+	active = TRUE
+	if(ticks_left > 0)
+		playsound(src, 'sound/items/timer.ogg', 30, FALSE)
+		balloon_alert_to_viewers("[ticks_left]")
+		ticks_left--
+		addtimer(CALLBACK(src, PROC_REF(explosive_countdown), ticks_left), 1 SECONDS)
+		return
+
+	playsound(src, 'sound/effects/snap.ogg', 75, TRUE)
+	if(!ishuman(loc))
+		balloon_alert_to_viewers("dud...")
+		active = FALSE
+		return
+	var/mob/living/carbon/human/brian = loc
+	if(brian.get_item_by_slot(ITEM_SLOT_NECK) != src)
+		balloon_alert_to_viewers("dud...")
+		active = FALSE
+		return
+	balloon_alert_to_viewers(UNLINT("BOOM!!"))
```

Comment:
Funny but I don't think this is necessary.
We use balloon alerts as one of many forms of feedback... Brian's head flying off with an explosion sound and visual flash probably counts as sufficient feedback here 😅